conftest.py

```diff
@@ -8,6 +9,51 @@
 log = logging.getLogger(__name__)
 
 
+# class SensorInfo:
```

Comment:
I understand that you were following along the lecture but generally it's better to avoid committing commented code.